charts/spire/templates/spire-controller-manager-webhook.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if .Values.controllerManager.enabled }}
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: spire-controller-manager-webhook
```

Comment:
Can a webhook be in a namespace?

if yes add namespace here.

Reply:
These are cluster-scoped resources, and we cannot tie them to a namespace 👀